README.md

```diff
@@ -21,11 +21,11 @@ In short, ICA is cross chain writes while ICQ is cross chain reads.
 
 ### Definitions 
 
-- `Host Chain`: The chain where the query is sent. The host chain listens for IBC packets from a controller chain that contain instructions (e.g. cosmos SDK messages) that the interchain account will execute.
-- `Querier Chain`: The chain sending the query to the host chain. The controller chain sends IBC packets to the host chain to query information.
+- `Host Chain`: The chain where the query is sent. The host chain listens for IBC packets from a querying chain that contain instructions (e.g. cosmos SDK messages) that the interchain account will execute.
+- `Querier Chain`: The chain sending the query to the host chain. The querying chain sends IBC packets to the host chain to query information.
```

Comment:
We define the chain sending the query to the host chain as the `Querier Chain` but then everywhere else it is referenced as the `Querying Chain`, it feels like these references should all use the same verbiage.